api/v1/cluster_webhook.go

```diff
@@ -2398,14 +2411,27 @@ func (r *Cluster) validatePgFailoverSlots() field.ErrorList {
 
 	const hotStandbyFeedbackKey = "hot_standby_feedback"
 	hotStandbyFeedback, hasHotStandbyFeedback := r.Spec.PostgresConfiguration.Parameters[hotStandbyFeedbackKey]
+	var hotStandbyFeedbackActivated bool
+	if hasHotStandbyFeedback {
+		var err error
+		hotStandbyFeedbackActivated, err = postgres.ParsePostgresBoolean(hotStandbyFeedback)
+		if err != nil {
+			result = append(
+				result,
+				field.Invalid(
+					field.NewPath("spec", "postgresql", "parameters", hotStandbyFeedbackKey),
+					hotStandbyFeedback,
+					fmt.Sprintf("invalid %s value. Must be a postgres boolean", hotStandbyFeedbackKey)))
```

Comment:
I would suggest to change with 
`fmt.Sprintf("invalid `%s` value. Must be a postgres boolean", hotStandbyFeedbackKey)))`

So that the log format looks similar to the one at line 1208

Reply:
Applied